### PR TITLE
fix Toluz + various minor tweaks (mostly AI hints)

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt [hunk 1]
```diff
@@ -8,6 +8,10 @@ T:Mode$ DiscardedAll | ValidPlayer$ You | ValidCard$ Card | TriggerZones$ Battle
 SVar:TrigExile:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card.TriggeredCards
 T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigReturn | TriggerDescription$ When NICKNAME dies, put the cards exiled with it into their owner's hand.
 SVar:TrigReturn:DB$ ChangeZoneAll | ChangeType$ Card.ExiledWithSource | Origin$ Exile | Destination$ Hand
+T:Mode$ ChangesZone | Origin$ Exile | Destination$ Any | Static$ True | ValidCard$ Card.IsRemembered | Execute$ DBForget
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt:11]: That looks wrong , the card doesn't even use remembering?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt]: 🤦‍♂️ let's try again and add that

File: forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt [hunk 2]
```diff
@@ -5,9 +5,13 @@ PT:3/1
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigConnive | TriggerDescription$ When CARDNAME enters the battlefield, it connives. (Draw a card, then discard a card. If you discarded a nonland card, put a +1/+1 counter on this creature.)
 SVar:TrigConnive:DB$ Connive
 T:Mode$ DiscardedAll | ValidPlayer$ You | ValidCard$ Card | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Whenever you discard one or more cards, exile them from your graveyard.
-SVar:TrigExile:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card.TriggeredCards
+SVar:TrigExile:DB$ ChangeZoneAll | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card.TriggeredCards | RememberChanged$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt:8]: Hmn ideally Remembered on ExiledWithSource would be redundant, I think it should have worked with it alone?

Maybe it's missing timestamp check?

Comment (Northmoc) [reply forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt:8]: There are many cards that use this janky way of doing it.

Probably is related to timestamps... the issue iirc is that when you bounce something, its "exiled with" doesn't get cleared

Comment (Northmoc) [reply forge-gui/res/cardsfolder/t/toluz_clever_conductor.txt:8]: @Hanmac has mentioned a few times that he has a refactor of how Forge handles exiling in the works

**Review by Agetian — APPROVED** [review]

**Pull request merged**